<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParticipantEditor</class>
 <widget class="QWidget" name="ParticipantEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1034</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel {
	border: 0.05em solid transparent;
	padding: 0.25em;
	font-size: 14pt;
}

QLineEdit {
        border: 0.05em solid black;
        border-radius: 0.25em;
        padding: 0.25em;
        font-size: 14pt;
        outline: none;
}
QLineEdit:focus {
        border: 0.05em solid #CA0B5E;
}
QLineEdit[error=true] {
        color: black;
        background-color: rgb(255, 230, 230);
}
QDateEdit {
        border: 0.05em solid black;
        border-radius: 0.25em;
        padding: 0.25em;
        font-size: 14pt;
        outline: none;
}
QDateEdit:focus {
        border: 0.05em solid #CA0B5E;
}
QDateEdit[error=true] {
        color: black;
        background-color: rgb(255, 230, 230);
}
QCheckBox {
        padding: 0.25em;
        font-size: 14pt;
        outline: none;
}
QCheckBox:focus {
        border: 0.05em solid #CA0B5E;
}
QCheckBox[error=true] {
        color: black;
        background-color: rgb(255, 230, 230);
}
QPushButton {
        border: 0.05em solid #CA0B5E;
        border-radius: 0.25em;
        color: #CA0B5E;
        padding: 0.5em;
        font-size: 14pt;
        outline: none;
}
QPushButton:hover,QPushButton:focus {
        background-color: rgba(202,11,94,0.8);
        color: white;
}
QPushButton:disabled {
        color: grey;
        border-color: grey;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QWidget" name="widget_2" native="true">
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="nameLabel">
          <property name="text">
           <string>name</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QWidget" name="nameWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,0,1">
           <property name="spacing">
            <number>10</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLineEdit" name="firstnameInput">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="focusPolicy">
              <enum>Qt::StrongFocus</enum>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="frame">
              <bool>false</bool>
             </property>
             <property name="placeholderText">
              <string>first name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="tussenvoegselsInput">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>75</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="frame">
              <bool>false</bool>
             </property>
             <property name="placeholderText">
              <string>tussenvoegsels</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lastnameInput">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="frame">
              <bool>false</bool>
             </property>
             <property name="placeholderText">
              <string>last name</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="dateOfBirthLabel">
          <property name="text">
           <string>date of birth</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="DateEditor" name="dateOfBirthInput" native="true">
          <property name="focusPolicy">
           <enum>Qt::WheelFocus</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QCheckBox" name="isTestParticipant">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>This is a test participant</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="confirmButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>confirm</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancelButton">
          <property name="text">
           <string>cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DateEditor</class>
   <extends>QWidget</extends>
   <header location="global">dateeditor.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>firstnameInput</tabstop>
  <tabstop>tussenvoegselsInput</tabstop>
  <tabstop>lastnameInput</tabstop>
  <tabstop>dateOfBirthInput</tabstop>
  <tabstop>isTestParticipant</tabstop>
  <tabstop>confirmButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
